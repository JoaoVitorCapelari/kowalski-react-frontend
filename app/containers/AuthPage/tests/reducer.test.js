import authPageReducer from '../reducer';

describe('authPageReducer', () => {
  it('returns the initial state', () => {
    expect(authPageReducer(undefined, {})).toMatchSnapshot();
  });
});
