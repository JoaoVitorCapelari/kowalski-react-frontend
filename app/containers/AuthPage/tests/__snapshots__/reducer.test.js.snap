// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`authPageReducer returns the initial state 1`] = `
Immutable.Map {
  username: "",
  password: "",
  isSubmitting: false,
  serverErrorMsg: "",
}
`;
